lib/transactions.js

```diff
@@ -276,13 +314,26 @@ exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAdd
             }
         // no founders rewards :)
         } else {
+            /*
+            Note: For ANON coin, it enters this code when fullnode doesn't return any masternode payee
+            This should never happen on mainnet, since there are plenty of masternodes. But, it is possible on ANON testnet, since there are not so many masternodes.
+            */
+
+            // this prevents NaN error
+            feeReward = feeReward || 0;
+            feePercent = feePercent || 0;
+            masternodeReward = masternodeReward || 0;
+            
+
             // pool t-addr
             txb.addOutput(
                 scriptCompile(poolAddrHash),
                 Math.round(blockReward.total * (1 - (feePercent / 100))) + feeReward - masternodeReward
             )
-
-            //masternode reward
+            
+            // masternode reward
+            // what if there is no masternode winner?
+            if(masternodeAddrHash)
```

Comment:
Can you fix the alignment of this block and add curly braces?

Reply:
Fixed.